integration/src/main/kotlin/kotlinx/benchmark/integration/ProjectBuilder.kt

```diff
@@ -34,8 +32,13 @@ benchmark {
 private val buildScript = run {
     """
     buildscript {
+        repositories {
+            mavenCentral()
+            maven { url '${System.getProperty("plugin_repo_url")}' }
+        }
         dependencies {
-            classpath files(${readFileList("plugin-classpath.txt")})
+            classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.8.21'
```

Comment:
The Kotlin version should be passed as a system property here as well. Because integration tests are executed in aggregate builds as well.